bark/models/behavior/idm/base_idm.cpp

```diff
@@ -39,12 +39,12 @@ BaseIDM::BaseIDM(const commons::ParamsPtr& params) : BehaviorModel(params) {
                       "See Wikipedia IDM article", 1.5f);
   param_max_acceleration_ = params->GetReal(
       "BehaviorIDMClassic::MaxAcceleration", "See Wikipedia IDM article", 1.7f);
-  param_acceleration_lower_bound_ =
-      params->GetReal("BehaviorIDMClassic::AccelerationLowerBound",
-                      "See Wikipedia IDM article", -5.0f);
-  param_acceleration_upper_bound_ =
-      params->GetReal("BehaviorIDMClassic::AccelerationUpperBound",
-                      "See Wikipedia IDM article", 8.0f);
+  // param_acceleration_lower_bound_ =
```

Comment:
should we not maintain both options:
- one where we restrict the accelerations as we desire
- and one where we strictly adhere to the RSS accelerations

I think especially for the other vehicles this could be interesting if not everyone behaves strictly according to the rss parameters (mixed-traffic).